.github/pull_request_template.md

```diff
@@ -15,8 +15,8 @@ Reference any related issues here.
 Before merging, the reviewer should check the following items:
 
 - [ ] Does the PR do what it claims to do?
-- [ ] If this is a new adaptor, has the migration tool been run and the
       migration guide followed?
+- [ ] If this is a new adaptor, Added the adaptor on marketing website ?
```

Comment:
You need to remove one more line here!

Reply:
@josephjclark that is what i added

Reply:
Since we are adding adaptor manually in the marketing website, i think this checkbox it's important for new adaptors

Reply:
I agree with the checkbox, I mean the line above, which is leftover from the previous edit.

I can't comment on it because it's not in the diff - this is the closest I can get.

Please re-read the guide carefully!